admin/schema.php

```diff
@@ -770,4 +770,12 @@ function installer_db_now() {
 
 # Release marker: 1.3.0-rc.1
 
+$g_upgrade[202] = array( 'AlterColumnSQL', array( db_get_table( 'api_token' ), "
+	user_id					I		UNSIGNED NOTNULL DEFAULT '0',
+	name					C(128)	NOTNULL DEFAULT \" '' \",
+	hash					C(128)	NOTNULL DEFAULT \" '' \",
+	date_created			I		UNSIGNED NOTNULL DEFAULT '1',
```

Comment:
I believe the 1 is historical, goes back to before the dates fields were converted to int (unix timestamps). I do agree it's kind of strange, but IMO maintaining consistency is reason enough.

I'll verify the API for the unused tokens check.

I'm not sure normalization is really needed if the above-mentioned check goes for <= 1 but I'll verify that too.